src/styles/Nav.scss

```diff
@@ -0,0 +1,129 @@
+.selected {
+  color: #0275b1 !important;
+}
+.icons > a,
+.user > a,
+.log-out > a {
+  text-decoration: none;
+  color: #181818;
+  &:hover,
+  &:focus,
+  &:active {
+    color: #0275b1;
+  }
+
+  & > p {
+    font-family: Euclid Circular B;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 8pt;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+}
+
+.nav {
+  background-color: white;
+  position: fixed;
+  justify-content: center;
+  top: 0;
+  width: 100%;
+}
+.navRow {
```

Comment:
a small thing but can we add a space after each `}` so it's slightly easier to read